### PR TITLE
[FEAT] Bills

### PR DESCRIPTION
**Pull request by SubhamPramanik**
This PR consists of:
- Getting bills
- Getting bill companies
- Paying bills

### CODE REVIEW

File: src/main/java/org/mifos/connector/gsma/account/BillsWorkers.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Value("${zeebe.client.evenly-allocated-max-jobs}")
+    private int workerMaxJobs;
+
+    @PostConstruct
+    public void setupWorkers() {
+
```
Comment (avikganguly01) [inline src/main/java/org/mifos/connector/gsma/account/BillsWorkers.java:47]: Function code is too long. Try to write small functions with less than 10 lines.

**Review by avikganguly01 — APPROVED** [review]

**Comment by avikganguly01** [comment]
Please rebase, resolve conflicts and squash commits.

File: src/main/java/org/mifos/connector/gsma/account/BillsWorkers.java (hunk 1 above)
Comment (SubhamPramanik) [reply src/main/java/org/mifos/connector/gsma/account/BillsWorkers.java:47]: This function consists of 2 different workers, it's similar to https://github.com/avikganguly01/ph-ee-connector-gsma/blob/master/src/main/java/org/mifos/connector/gsma/zeebe/ZeebeeWorkers.java

Comment (avikganguly01) [reply src/main/java/org/mifos/connector/gsma/account/BillsWorkers.java:47]: No matter the conditions, you need to find a way to write maintainable functions and this function might work but it isn't maintainable.

**Pull request merged**